sdk/communication/communication-job-router/review/communication-job-router.api.md

```diff
@@ -0,0 +1,1491 @@
+## API Report File for "@azure/communication-job-router"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference lib="esnext.asynciterable" />
+
+import * as coreHttp from '@azure/core-http';
+import { KeyCredential } from '@azure/core-auth';
+import { OperationOptions } from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PipelineOptions } from '@azure/core-http';
+import { RequestPolicyFactory } from '@azure/core-http';
+import { RestResponse } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public (undocumented)
+export interface AcceptJobOfferResponse {
+    assignmentId: string;
+    jobId: string;
+    workerId: string;
+}
+
+// @public
+export interface AcceptJobOptions extends JobRouterAcceptJobOptionalParams {
+}
+
+// @public
+export type AzureFunctionRule = RouterRule & {
+    kind: "azure-function-rule";
+    functionAppUrl: string;
+    functionName: string;
+    credential?: AzureFunctionRuleCredential;
+};
+
+// @public
+export interface AzureFunctionRuleCredential {
+    appKey?: string;
+    clientId?: string;
+    functionKey?: string;
+}
+
+// @public
+export type BestWorkerMode = DistributionMode & {
+    kind: "best-worker";
+    scoringRule?: RouterRuleUnion;
+    scoringRuleOptions?: ScoringRuleOptions;
+};
+
+// @public
+export type CancelExceptionAction = ExceptionAction & {
+    kind: "cancel";
+    note?: string;
+    dispositionCode?: string;
+};
+
+// @public
+export interface CancelJobOptions extends JobRouterCancelJobOptionalParams {
+}
+
+// @public
+export interface CancelJobRequest {
+    dispositionCode?: string;
+    note?: string;
+}
+
+// @public
+export interface ChannelCollection {
+    readonly nextLink?: string;
+    // (undocumented)
+    value: RouterChannel[];
+}
+
+// @public (undocumented)
```

Comment:
Let’s document all the undocumented things 🙂 if it helps, you can make an issue and do it outside of this PR.

Reply:
sure, can you tell me what need to be documented? I compare this api file with other sdk, didn't find difference.